docs/contributing/contributing.rst

```diff
@@ -310,6 +311,31 @@ with ``worker`` to get shell access in your worker.
     if Docker can't find the file/directory on the host it will silently fail and mount
     in an empty directory.
 
+
+.. admonition:: Enabling FUSE
+
+    When running toil-wdl-runner with Singularity, Singularity will sandbox images
```

Comment:
This sounds like we mean it will restrict what the images are allowed to do, but we really mean that it will decompress the image files that store the container image filesystems and turn them into built-out directories full of files, which it calls "sandbox directories".

```suggestion
    When running toil-wdl-runner with Singularity, Singularity will decompress images to sandbox directories
```